com.unity.render-pipelines.universal/Editor/ShaderPreprocessor.cs

```diff
@@ -613,6 +975,11 @@ private static ShaderFeatures GetSupportedShaderFeatures(UniversalRenderPipeline
                     {
                         rendererClustered = universalRendererData.renderingMode == RenderingMode.Forward &&
                             universalRendererData.clusteredRendering;
+
+#if ENABLE_VR && ENABLE_XR_MODULE
+                        if (universalRendererData.xrSystemData != null)
+                            shaderFeatures |= ShaderFeatures.DrawProcedural;
+#endif
```

Comment:
Currently it seems like draw procedural is only used with VR, so I strip based on that now. Even if plan to use it in non VR I think we should have somekind of option, do not see a point of having variants here.